src/assets/RealAsset/RealAsset/realAsset.test.js

```diff
@@ -0,0 +1,38 @@
+import RealAsset from './realAsset.js'
+
+describe('RealAsset', () => {
+  describe('constructor', () => {
+      it('set ownership correctly', ()=>{
+        const ownerships={
+            ownership1:{
+                partyId: 1,
+                split: 0.5
+            },
+            ownerhsip2:{
+                partyId: 2, 
+                split: 0.5
+            }
+        }
+       const testRealAsset=new RealAsset({assetId: 'assetId', ownership: ownerships})
+       expect(testRealAsset.ownership).toEqual(ownerships)
+      })
+
+      it('should throw an error message', ()=>{
+        const ownerships={
```

Comment:
Thanks. I see. And I just noticed that I didn't do partyId check, I will add partyId check too.